README.md

```diff
@@ -205,6 +209,55 @@ Retriable.retriable on_retry: do_this_on_each_retry do
 end
 ```
 
+### RetriableContexts
+
+There is a separate plugin to enable the contexts feature.  It's not included by default; to use it do:
+
+```ruby
+require 'retriable_contexts'
```

Comment:
Can we possibly move this into the `retriable` namespace? So you'd require it like:

```ruby
require 'retriable/contexts`
```

Similar to:

```ruby
require 'retriable/core_ext/kernel'
```

I think it would be better to have all extensions under one namespace rather then define their own, especially if more extensions get added in the future.